plugins/obs-outputs/librtmp/rtmp.c

```diff
@@ -314,42 +314,38 @@ RTMP_TLS_LoadCerts(RTMP *r) {
     CertFreeCertificateContext(pCertContext);
     CertCloseStore(hCertStore, 0);
 #elif defined(__APPLE__)
-    SecKeychainRef keychain_ref;
-    CFMutableDictionaryRef search_settings_ref;
-    CFArrayRef result_ref;
+#elif defined(__APPLE__)
```

Comment:
Doesn't this make two `elif defined(__APPLE__)` in a row?

Reply:
That is indeed an oversight.